inc/customizer/class-storefront-customizer.php

```diff
@@ -817,6 +819,82 @@ public function get_css() {
 			return apply_filters( 'storefront_customizer_css', $styles );
 		}
 
+		/**
+		 * Get Gutenberg Customizer css.
+		 *
+		 * @see get_storefront_theme_mods()
+		 * @return array $styles the css
+		 */
+		public function gutenberg_get_css() {
+			$storefront_theme_mods = $this->get_storefront_theme_mods();
+			$brighten_factor       = apply_filters( 'storefront_brighten_factor', 25 );
```

Comment:
Unused variable. I'm not sure if it should be removed or if it should be used in a similar fashion as `$darken_factor` is used.